taipy/gui/viselements.json

```diff
@@ -1494,7 +1494,7 @@
                     },
                     {
                         "name": "adapter",
-                        "type": "Function",
+                        "type": "Callable",
```

Comment:
Shouldn't this be `Union[str, Callable]`?
In this case, there are plenty of other doc locations you get get the text from.